mantidproject/mantid
# 15574 plot every error bar

Performs a one off change on first run to plot every error bar.

This is just a change to a setting, subsequent manual changes should be preserved.

**To test:**
1. Load a dataset, plot with errors, zoom in close, every point should have an error bar.
2. Then use view -> preferences 2D plots->curves, set draw all errors off, and do the same thing.
3. Look at the Release notes and check they make sense.

Fixes #15574

[UI Release notes](/docs/source/release/v3.7.0/ui.rst)



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
